ported_arrows/stress/send_token_from_minter.py

```diff
@@ -33,19 +32,16 @@ def main(cli_args: List[str]):
 
     for account in accounts.get_all():
 
-        transaction = Transaction(
-            chain_id=network.chain_id,
-            sender=minter.address.bech32(),
-            receiver=account.address.bech32(),
-            gas_limit=250000 + 50000 + 1500 * len(transaction.data),
-        )
+        send_config = DefaultTransactionBuildersConfiguration(network.chain_id)
+        payments = [TokenPayment.fungible_from_integer(args.token, args.amount_atoms, 18)]
+        transaction = MultiESDTNFTTransferBuilder(
```

Comment:
why don;t you use single ESDT transfer?

Reply:
Habit of using MultiESDTNFTTransferBuilder as a generic hammer for everything that looks like a nail, since this can easily solve both fungibles and nonfungibles without the need to have different handlers for each.
I know that gas consumption is smaller in the single token transfer operation, but in my opinion the difference is so marginal that at this point it doesn't make much sense to bother using each specific single transfer function.